### ISSUE TITLE
Alcoholic ketoacidosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5871]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

Proposed Definition: A metabolic acidosis due to accumulation of ketone bodies in setting of alcohol intake
--

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@avizrosenberg
@yongqunh  
@beckysteck

HPO terms try not to encode etiology, because then we would get a combinatorial explosion (there are multiple etiologies for almost all of the 14000+ HPO terms!). 

I think this is better conceptualized as an acute disease and therefore should go into MONDO (@nicolevasilevsky )
See also https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2564331/

**Comment by pnrobinson** [comment]
see https://github.com/monarch-initiative/mondo/issues/1823
